tests/wpunit/Common/Libraries/__snapshots__/Uplink_Controller_Test__it_should_setup_license_fields__0.snapshot.html

```diff
@@ -0,0 +1,29 @@
+
+	<div class="stellarwp-uplink__license-field">
```

Comment:
Why this file? Was it originally missing?

Reply:
Yes it was and it seems like we need it.